app/src/components/Dialog.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
+    <md-dialog-confirm
+      :md-active.sync="active"
+      :md-title="dialogTitle"
+      :md-content="textContent"
+      :md-confirm-text="confirmTextBtn"
+      :md-cancel-text="cancelTextBtn"
+      @md-cancel="onCancel"
+      @md-confirm="onConfirm" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Dialog',
+  props: {
+    active: {
+      type: Boolean,
+      default: false
+    },
+    cancelTextBtn: {
+      type: String,
+      default: 'Disagree'
+    },
+    confirmTextBtn: {
+      type: String,
+      default: 'Agree'
+    },
+    textContent: {
+      type: String,
+      default: 'Welcome to <strong>Materialsmine</strong>.<br /> Here is a reusable dialog box in test mode'
+    },
```

Comment:
I'm wondering if would make sense to use <slot>s for the dialog box content (for the ability to have custom html) rather than passing just textContent as a prop

Reply:
@stouffers also makes some suggestions. I'll redo it